libs/datasets/combined_datasets.py

```diff
@@ -196,29 +196,24 @@ def datasource_regions(
     Region.from_fips("02060"),
     # Culebra Municipio, PR - reports 99.9% of 12+ population vaccinated [7/2021]
     Region.from_fips("72049"),
+    # Chattahoochee County, GA - reports well over 100% 1+ dose vaccinated [2/2022]
+    Region.from_fips("13053"),
```

Comment:
If we'd prefer to not mix sources within a state/region we can remove these county exclusions and potentially look to block these locations instead

Reply:
I'm a bit on the fence.  Let's go ahead with keeping them for now, I guess.  I get more nervous when there are a lot of regions in a state or they're the biggest counties.  But small one-off counties 🤷